src/crystal/system/unix/mime.cr

```diff
@@ -0,0 +1,24 @@
+module Crystal::System::MIME
+  MIME_SOURCES = [
+    "/etc/mime.types",
+    "/etc/apache2/mime.types",
+    "/etc/apache/mime.types",
```

Comment:
Why looking at `/etc/apache` and `/etc/apache2`?! The mimes sources should be standard locations, and be platform  and software agnostic (here its specific to Apache on Debian). For instance, widely used options like Red Hat derivatives use `/etc/httpd`, and Nginx has its mime types in another location.
Furthermore on FreeBSD, the location is `/usr/local/etc/apache2x/`.

Reply:
I just followed Go's example here (see [Go docs](https://golang.org/pkg/mime/#TypeByExtension)). Obviously, this list can easily be extended. Maybe like Python's `knownfiles`:

https://github.com/python/cpython/blob/bc9aa813a34474e517af8999565ff6151559d42f/Lib/mimetypes.py#L42-L52